src/components/CheckboxInput/CheckboxInput.module.css

```diff
@@ -1,71 +1,64 @@
 @import '../../design-tokens/mixins.css';
 
-/*------------------------------------*\
-    # CHECKBOX INPUT
-\*------------------------------------*/
-
-/**
- * Wraps the visually hidden checkbox input element and the visible sibling svg element.
- */
-.input__wrapper {
+.wrapper {
   position: relative;
   /* Centers the checkbox icon in the wrapper. */
   display: inline-flex;
   align-items: center;
   /* Aligns the checkbox with the first line of the label. */
   align-self: flex-start;
 }
-/**
- * The visually hidden input element for the checkbox. The visual checkmark is provided by an svg element.
- */
+
 .checkbox__input {
-  cursor: pointer;
-  /* Removes default margins placed by browser for checkboxes. */
+  height: 18px;
+  width: 18px;
+
+  /* Remove the browser's checkbox styles, allowing us to provide our own. */
+  appearance: none;
   margin: 0;
-  /* Remove the checkbox from the page flow, positioning it on top of the SVG. */
-  position: absolute;
-  left: 0.25rem;
-  /* Set same dimensions as the CheckboxSvg element. */
-  height: var(--eds-size-2);
-  width: var(--eds-size-2);
-  /** 
-   * Hide the input element visually.
-   * This ensures the checkbox is still "physically" present so that all users,
-   * especially on touch screen readers, still interact with the real checkbox element
-   * where it would naturally be present. 
-   */
-  opacity: 0;
-}
 
-/**
- * The visible checkbox svg icon element.
- */
-.checkbox__icon {
   color: var(--eds-theme-color-icon-brand-primary);
+  border: 2px solid currentColor;
+
+  /* Place the ::before content right in the middle of the input. */
+  display: grid;
+  place-content: center;
+}
+
+.checkbox__input:checked::before {
+  background-color: currentColor;
+  content: '';
+  height: 24px;
+  width: 24px;
+
+  clip-path: path("M10.6 16.2L17.65 9.15L16.25 7.75L10.6 13.4L7.75 10.55L6.35 11.95L10.6 16.2ZM3 21V3H21V21H3Z");
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path

Will need to slightly update our browser support

Reply:
Also, kinda tempted to put all these styles behind an `@supports (clip-path: path)` and gracefully fall back to `accent-color`.

Reply:
@ahuth is the clip-path the SVG for the checkmark itself?

Reply:
That's right 👍